docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -250,9 +253,36 @@ your installation path. You can use the following settings, if you installed R a
 - Windows: `C:\Program Files\R\R-3.6.X'` (where X is the version you used to install the above libraries)
 - macOS: `/Library/Frameworks/R.framework/Versions/3.6/Resources`
 
+##### KNIME OpenMS plugin
+
 You are now ready to install the OpenMS nodes.
-- Open KNIME.
-- Click on **Help** > **Install New Software**
+- In KNIME, click on **Help** > **Install New Software**
+
+You now have to choose an _update site_ to install the OpenMS plugin from. Which _update site_ to choose depends on if you received an USB stick
+in a hands-on Tutorial or if you are doing this Tutorial online.
+
+::::{tab-set}
+
+:::{tab-item} Online
+:sync: online
+
+To install the OpenMS KNIME plugin from the internet, do the following:
+
+1. From the '**Work with**:' drop-down list, select the _update site_ 'KNIME Community Extensions (Trusted) - https://update.knime.com/community-contributions/trusted/4.6'
```

Comment:
https://update.knime.com/community-contributions/trusted/4.6/.   when i open this link it gives me permission error 403 forbidden.